src/main/java/com/google/api/codegen/config/MethodConfig.java

```diff
@@ -280,9 +283,9 @@ public GrpcStreamingType getGrpcStreamingType() {
     }
   }
 
-  /** Returns true if this method has flattening configured. */
+  /** Returns true if this method supports parameter flattening. */
   public boolean isFlattening() {
-    return flattening != null;
+    return !isGrpcStreaming() && flattening != null;
```

Comment:
Wait, so will this silently ignore flattening configuration if it is a grpc streaming method? If we don't want to allow it, we should emit an error somewhere instead of hiding it.